cpp/src/cache/image_cache_config.cpp

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2021, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "cucim/cache/image_cache_config.h"
+
+#include <fmt/format.h>
+#include <nlohmann/json.hpp>
+
+using json = nlohmann::json;
+
+namespace cucim::cache
+{
+
+void ImageCacheConfig::load_config(void* json_obj)
```

Comment:
Made it to `const`.
However, doesn't want to declare it with `json&` because we don't want to expose `json` type to the public header (https://github.com/gigony/cucim/blob/566f130425d2c4783788776c832d15783b4f665d/cpp/include/cucim/cache/image_cache_config.h#L43) which requires either cucim library package `nlohmann::json` header or application code depends on it.